src/components/DataInput/DataInput.cy.tsx

```diff
@@ -167,9 +167,13 @@ describe("Data Input Components", () => {
             cy.get("li[data-value='a']").click();
             cy.get("@onChangeSpy").should("have.been.calledWith", "a");
 
+            cy.get("div[class^='MuiSelect']").click();
+
             cy.get("li[data-value='b']").click();
             cy.get("@onChangeSpy").should("have.been.calledWith", "b");
 
+            cy.get("div[class^='MuiSelect']").click();
```

Comment:
So my understanding of this is wrong I think, but I thought this div targeted like the header bit of the dropdown menu. If this is the case, would this not open and shut the menu? Does seem to work though so I think I'm wrong.